tests/thread_float/Makefile

```diff
@@ -0,0 +1,16 @@
+APPLICATION = thread_float
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon calliope-mini cc2650stk chronos maple-mini \
```

Comment:
Seems like murdock is still not happy, please add the following boards:
nucleo-f031k6
nucleo-f042k6

than maybe squash and everything should be good.